src/trumbowyg.js

```diff
@@ -338,7 +338,8 @@ jQuery.trumbowyg = {
                 .addClass(prefix + 'editor')
                 .attr({
                     'contenteditable': true,
-                    'dir': t.lang._dir || t.o.dir
+                    'dir': t.lang._dir || t.o.dir,
+                    'tabindex': t.tabindex || -1
```

Comment:
I'm not sure about that. By default, we don't want this attribute. Add it only if exists.

I maybe fix that ASAP or if you want, you can do it :)